notify-jira-about-contribution/action.yml

```
@@ -47,85 +47,100 @@ runs:
-        old_major=$(echo "$old_version" | cut -d. -f1)
-        new_major=$(echo "$new_version" | cut -d. -f1)
-
-        isMajorUpdate=false
-        if [ "$old_major" != "$new_major" ]; then
-          isMajorUpdate=true
+        isDraft=${{ env.IS_DRAFT }}
+        if [[ $isDraft == "true" ]]; then
+          echo "Pull request is a draft"
+          echo "result=false" >> $GITHUB_OUTPUT
+          exit
         fi
 
-        isTeamMember=${{ env.IS_TEAM_MEMBER }}
-        isDependabot=${{ env.IS_DEPENDABOT }}
-        notifyAboutMajorDependabotUpdates=${{ env.NOTIFY_ABOUT_MAJOR_DEPENDABOT_UPDATES }}
-        if [[ $notifyAboutMajorDependabotUpdates == "true" && $isDependabot == "true" ]]; then
-          if [[ $isMajorUpdate == "true" ]]; then
-            echo "Major dependabot update, exiting"
-            echo "result=true" >> $GITHUB_OUTPUT
-          else
-            echo "Non-major dependabot update"
+        # Exit early if pull request is not from dependabot
+        isDependabotPullRequest=${{ env.IS_DEPENDABOT_PULL_REQUEST }}
+        if [[ $isDependabotPullRequest == "false" ]]; then
+          isTeamMember=${{ env.IS_TEAM_MEMBER }}
+          if [[ $isTeamMember == "true" ]]; then
+            echo "Pull request author is member of ${{ inputs.team }}"
             echo "result=false" >> $GITHUB_OUTPUT
+            exit
           fi
 
+          echo "Pull request author is not a member of ${{ inputs.team }}"
+          echo "result=true" >> $GITHUB_OUTPUT
           exit
         fi
 
-        if [[ $isTeamMember == "false" ]]; then
-          echo "Author is not a member of the team, exiting"
-          echo "result=true" >> $GITHUB_OUTPUT
+        shouldNotifyAboutMajorDependencyUpdates=${{ env.SHOULD_NOTIFY_ABOUT_MAJOR_DEPENDENCY_UPDATES }}
+        if [[ $shouldNotifyAboutMajorDependencyUpdates == "false" ]]; then
+          echo "No need to notify about dependency updates"
+          echo "result=false" >> $GITHUB_OUTPUT
+          exit
+        fi
+
+        pullRequestTitle="${{ env.PR_TITLE }}"
+
+        # Extract major versions using grep utility
+        # For "Bump @cypress/webpack-preprocessor from 5.17.1 to 6.0.1" it returns "5"
+        currentMajorVersion=$(echo "$pullRequestTitle" | grep --only-matching --extended-regexp 'from [0-9]+(\.[0-9]+)?' | cut -d' ' -f2 | cut -d'.' -f1)
+        if [ -z "$currentMajorVersion" ]; then
+          echo "Error: Unable to extract current major version" >&2
+          exit 1
+        fi
+
+        # For "Bump @cypress/webpack-preprocessor from 5.17.1 to 6.0.1" it returns "6"
+        newMajorVersion=$(echo "$pullRequestTitle" | grep --only-matching --extended-regexp 'to [0-9]+(\.[0-9]+)?' | cut -d' ' -f2 | cut -d'.' -f1)
+        if [ -z "$newMajorVersion" ]; then
+          echo "Error: Unable to extract new major version" >&2
+          exit 1
+        fi
+
+        if [ "$currentMajorVersion" == "$newMajorVersion" ]; then
+          echo "Is not a major dependency update"
```

```suggestion
          echo "It's minor or patch update. We don't create tickets for those. Exiting."
```
Lets be positive? :)

---

@OleksandrNechai Added, thanks!